app/test/timeouts_test.go

```diff
@@ -0,0 +1,92 @@
+package app_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/v3/pkg/appconsts"
+	"github.com/celestiaorg/celestia-app/v3/test/util/testfactory"
+	"github.com/celestiaorg/celestia-app/v3/test/util/testnode"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/celestiaorg/celestia-app/v3/app"
+	"github.com/celestiaorg/celestia-app/v3/app/encoding"
+	"github.com/celestiaorg/celestia-app/v3/pkg/user"
+)
+
+func TestTimeoutTestSuite(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping app/test/timeouts_test in short mode.")
+	}
+	suite.Run(t, &TimeoutTestSuite{})
+}
+
+type TimeoutTestSuite struct {
+	suite.Suite
+
+	ecfg     encoding.Config
+	accounts []string
+	cctx     testnode.Context
+}
+
+const (
+	timeoutPropose = 7 * time.Second
```

Comment:
Should these be the same as below? in pkg/appconsts/v3/app_consts.go

<img width="420" alt="Screenshot 2024-10-10 at 15 39 46" src="https://github.com/user-attachments/assets/40f98e33-478a-4589-b27d-1407eca6b203">

Reply:
No, they shouldn't, the reason is that these are the initial values configured for a node, intentionally set to differ from the default versioned timeouts to verify that nodes derive timeouts from the app version rather than the config settings.

Reply:
nice, thank you for explaining!